playground/frontend/playground_components/lib/src/playground_components.dart

```diff
@@ -26,4 +30,11 @@ class PlaygroundComponents {
     packageName,
     path: 'assets/translations',
   );
+
+  static Future<void> ensureInitialized() async {
+    await initializeServiceLocator();
+  }
+
+  static SymbolsNotifier get symbolsNotifier =>
+      GetIt.instance.get<SymbolsNotifier>();
```

Comment:
Are we definitely have to add new DI system in our app? Why can't we use provider for the same goal?

Reply:
This is more of a service locator than a dependency injection. SL's objects are a form of global variables while provider is a mean to make something available to a subtree.

Provider should be used if it makes sense to override an object to certain parts of the tree. For instance, we do this with themes in 'Share my code' button.

For objects that are global by design, hanging them with provider adds complexity to their management. Also with SL objects are available in other business logic objects and not only in widgets.

`SymbolsNotifier` will be used in `SnippetEditingController`. Passing it though widgets is way more complex than just using a global instance with `get_it`.